EIPS/eip-4844.md

```diff
@@ -267,33 +271,48 @@ def point_evaluation_precompile(input: Bytes) -> Bytes:
 
 ### Gas price of blobs (Simplified version)
 
-For early draft implementations, we simply change `get_blob_gas(parent)` to always return `GAS_PER_BLOB`.
+***WARNING: This is only for testing***
 
-### Gas price update rule (Full version)
+For early draft implementations, we simply change `get_blob_gas(parent)` to always return `SIMPLE_GAS_PER_BLOB`.
```

Comment:
EIPs should be written toward their final target and not contain historical details (one can look at commit history and discussions for that).  It is OK to not be fully committed to everything in the EIP yet, but the *form* of the EIP should target its final state (which means not saying things like "For early draft implementations do X instead of Y".  Instead, just say "Do X" and then later that may change to "Do Y".

That being said, I'm not an EIP editor anymore so I have no actual authority over this matter.

Reply:
Given that this PR does not introduce either of the [simplified/full] sections, but just modifies the full one, I think removing the simplified one should not be in scope for the PR. It does seem reasonable to me to remove it soon though.